### PR TITLE
Restructure files for scss

### PR DESCRIPTION
**Pull request by foxwellm**
I had to "npm install node-sass --save" to get the sass package, not sure if you will have to type or if it will already work once you pull. 

I moved all the new .SCSS files to their own folder so that Main.scss can pull them all easier using the _ in front of the names. That meant there was no point in having the individual folders anymore for each component, so took each of them out. 

I was worried about merge conflicts because I'm moving files around, so I pulled Justin's destsructure branch first, so his changes are there too. All you have to worry about for this PR is the file structure.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,102 +1,105 @@
 import React, { Component } from 'react';
-import './App.css';
-import Navbar from './Components/Navbar';
-import LandingPage from './Components/LandingPage';
-import SearchPage from './Components/SearchPage';
+import './styles/Main.scss';
```
Comment (foxwellm) [inline src/App.js:2]: Main contains all references to scss files

File: src/styles/_Featured.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
 .Featured {
-  height: 60%;
+  height: 500px;
```
Comment (foxwellm) [inline src/styles/_Featured.scss]: % was not acting as intended after the file restructure.

File: src/styles/_Navbar.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
 .Navbar {
-  height: 10%;
+  height: 100px;
```
Comment (foxwellm) [inline src/styles/_Navbar.scss]: % was not acting as intended after the file restructure.

File: src/App.js (hunk 1 above)
Comment (JustinD85) [reply src/App.js:2]: I noticed App.css wasn't converted. Was that intentional?

File: src/styles/_Featured.scss (hunk 2 above)
Comment (JustinD85) [reply src/styles/_Featured.scss]: I believe you are not including App.css in the conversion and thus there was no parent with styling. Could you convert App.css as well and see if that solves this issue?

**Comment by JustinD85** [comment]
issue #18

File: src/App.js (hunk 1 above)
Comment (foxwellm) [reply src/App.js:2]: No it wasn't. Good catch

Comment (foxwellm) [reply src/App.js:2]: App.css have now been moved to styles, added to Main.scss and renamed to App.scss

**Review by JustinD85 — APPROVED** [review]

**Review by whitneyfirth — APPROVED** [review]

**Pull request merged**